### ISSUE TITLE
Several items in the production BOM are no longer in stock

### ISSUE THREAD

**Issue by bragdonjm** [othermod/PSPi-Version-6#13]
Several components show shortfall attempting to load in the BOM + Placement files. There doesnt seem to be any option to wait for restock or estimated times for restock. There is an option, however, to replace components but their interface shows zero alternatives as far as I can tell. Figured I would float this up to you.

**Comment by othermod** [comment]
Yeah I've been playing whack a mole with alternative parts for the last few prototypes. Most of the parts are pretty generic, except for the DC converters and audio amplifier.

**Comment by bragdonjm** [comment]
Cool - feel free to leave open for others that will potentially ask. I did order some stuff from Ali to get it on the way, probably wrong now. Let me know how else I can try to help. My skillset is mostly automation and devops - and I can code - not sure how that will help though.
